### PR TITLE
Update dialogueBox and text sizing. Simulate a Next button. Hide text after displaying once

### PR DESCRIPTION
**Pull request by toolness**

### CODE REVIEW

**Comment by joancc** [comment]
Update ready to merge. Will take a second look at dialogues once Adam goes over the story beginning to end.

File: mini/index.html [hunk 1]
```diff
@@ -26,6 +26,7 @@
     <div id="p5_canvas"></div>
     <button id="previous" onclick="previousLevel()" style="display: none">Previous Level</button>
     <button id="next" onclick="nextLevel()" style="display: none">Next Level</button>
+    <button id="next" onclick="showDialogue()">Story</button>
```
Comment (toolness) [inline mini/index.html]: Hmm the `id` of this button probably shouldn't be `next`?

Comment (joancc) [inline mini/index.html]: Def not. Just updated.

**Comment by toolness** [comment]
Woooooooooot thanks!

**Pull request merged**